scripts/deploy_assisted_service.sh

```diff
@@ -140,7 +151,13 @@ else
     fi
 
     skipper run src/update_assisted_service_cm.py
-    (cd assisted-service/ && skipper --env-file ../skipper.env run "make deploy-all" ${SKIPPER_PARAMS} $ENABLE_KUBE_API_CMD DEPLOY_TAG=${DEPLOY_TAG} DEPLOY_MANIFEST_PATH=${DEPLOY_MANIFEST_PATH} DEPLOY_MANIFEST_TAG=${DEPLOY_MANIFEST_TAG} NAMESPACE=${NAMESPACE} AUTH_TYPE=${AUTH_TYPE} ${DEBUG_DEPLOY_AI_PARAMS:-})
+    if [ $DEPLOY_TARGET == minikube ]; then
+        export TARGET=local  # in assisted-service deployment scripts, minikube is annotated as 'local'
+    else
+        export TARGET=$DEPLOY_TARGET
+    fi
```

Comment:
I don't like those parts very particularly 
Maybe the idea of using ``local`` as code-name for ``minikube`` was making it more generalized, but now when adding additional local k8s option that has different networking arch it isn't completely interchangeable
 
I might be able to change occurrences of ``local`` in ``minikube`` in assisted-service, or maybe some other option I haven't thought of?

Reply:
I guess `local` made sense when we only had one way to deploy locally.
Replacing it by `minikube` looks fine to me.

Reply:
Ack. Created https://github.com/openshift/assisted-service/pull/4642 to do the rename